src/library_gl.js

```diff
@@ -822,27 +821,32 @@ var LibraryGL = {
 #endif
 
       // Detect the presence of a few extensions manually, this GL interop layer itself will need to know if they exist. 
+#if LEGACY_GL_EMULATION
       context.compressionExt = GLctx.getExtension('WEBGL_compressed_texture_s3tc');
       context.anisotropicExt = GLctx.getExtension('EXT_texture_filter_anisotropic');
-      context.floatExt = GLctx.getExtension('OES_texture_float');
```

Comment:
The variable `context.floatExt` is not referenced anywhere in the code, so we don't need to store the return value anywhere. In the same `initExtensions` function below, `OES_texture_float` is included so it will be enabled there, which makes this line redundant.